### PR TITLE
feat: price too low for items

### PR DESCRIPTION
**Pull request by nicosantangelo**
https://user-images.githubusercontent.com/692077/167705844-336e957d-803d-4c6a-b95f-c12c57722181.mov

Related: https://github.com/decentraland/dapps-issues/issues/23

### CODE REVIEW

File: webapp/src/modules/translation/locales/en.json [hunk 1]
```diff
@@ -30,6 +30,7 @@
     "sale_type": "Sale type",
     "sell_price": "Sell price",
     "no_results": "No results",
+    "learn_more": "Learn More",
```
Comment (LautaroPetaccio) [inline webapp/src/modules/translation/locales/en.json:33]: This change seems to not have been replicated in the other languages.

File: webapp/src/components/BuyPage/utils.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+/**
+ * Checks against the min value of costless item sales
+ */
+export function isPriceTooLow(type: AssetType, asset: Asset) {
+  if (type === AssetType.ITEM) {
+    const { price } = asset as Item
+    const minSaleValue = getMinSaleValueInWei()
+    if (minSaleValue) {
+      return toBN(price).gte(toBN(minSaleValue))
```
Comment (LautaroPetaccio) [inline webapp/src/components/BuyPage/utils.ts:13]: Isn't it lower than?
```suggestion
      return toBN(price).lt(toBN(minSaleValue))
```

File: webapp/src/components/BuyPage/utils.spec.ts [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    beforeEach(() => {
+      process.env.REACT_APP_MIN_SALE_VALUE_IN_WEI = ''
+    })
+
+    it('should return true', () => {
+      expect(isPriceTooLow(AssetType.ITEM, {} as Item)).toBe(true)
+    })
+  })
+
+  describe('when there a min sale value', () => {
```
Comment (LautaroPetaccio) [inline webapp/src/components/BuyPage/utils.spec.ts:41]:
```suggestion
  describe('when there is a min sale value', () => {
```

File: webapp/src/components/BuyPage/utils.spec.ts [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+    beforeEach(() => {
+      process.env.REACT_APP_MIN_SALE_VALUE_IN_WEI = minSaleValue
+    })
+
+    describe('and the asset type is NFT', () => {
+      it('should return true', () => {
+        expect(isPriceTooLow(AssetType.NFT, {} as Item)).toBe(true)
+      })
+    })
```
Comment (LautaroPetaccio) [inline webapp/src/components/BuyPage/utils.spec.ts:52]: Are we just covering minting of new items with this change?

File: webapp/src/components/BuyPage/utils.spec.ts [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+    describe('and the asset type is ITEM', () => {
+      it('should return true if the price is greater than the minimum', () => {
+        expect(
+          isPriceTooLow(AssetType.ITEM, {
```
Comment (LautaroPetaccio) [inline webapp/src/components/BuyPage/utils.spec.ts:57]: The name might be a bit weird, if the price is too low, why is it returning true for a min price that is lower than the price? It seems that the price is higher than the minimum amount.

File: webapp/src/components/BuyPage/MintItemModal/MintItemModal.tsx [hunk 6]
```diff
@@ -77,6 +79,9 @@ const MintItemModal = (props: Props) => {
     setShowAuthorizationModal
   ])
 
+  const isInDifferentChainId = wallet.chainId !== item.chainId
+  const hasLowPrice =
+    !isPriceTooLow(AssetType.ITEM, item) && isInDifferentChainId
```
Comment (LautaroPetaccio) [inline webapp/src/components/BuyPage/MintItemModal/MintItemModal.tsx:84]: Isn't the condition `isPriceTooLow(AssetType.ITEM, item) && !isInDifferentChainId`?

**Review by LautaroPetaccio — CHANGES_REQUESTED** [review]

File: webapp/src/modules/translation/locales/en.json (hunk 1 above)
Comment (nicosantangelo) [reply webapp/src/modules/translation/locales/en.json:33]: Good eye!

File: webapp/src/components/BuyPage/utils.ts (hunk 2 above)
Comment (nicosantangelo) [reply webapp/src/components/BuyPage/utils.ts:13]: Correct, the function is actually fine, the refactored name was the problem. Changing tho!

File: webapp/src/components/BuyPage/MintItemModal/MintItemModal.tsx (hunk 6 above)
Comment (nicosantangelo) [reply webapp/src/components/BuyPage/MintItemModal/MintItemModal.tsx:84]: fixed, related to the rename discussed above

File: webapp/src/components/BuyPage/utils.spec.ts (hunk 5 above)
Comment (nicosantangelo) [reply webapp/src/components/BuyPage/utils.spec.ts:57]: fixed, related to the rename discussed above

**Review by LautaroPetaccio — CHANGES_REQUESTED** [review]

**Review by LautaroPetaccio — APPROVED** [review]

**Pull request merged**